lib/addons/cloudwatch-adot-addon/index.ts

```diff
@@ -12,7 +12,7 @@ import { KubectlProvider, ManifestDeployment } from "../helm-addon/kubectl-provi
  */
 
 /**
- * Configuration options for add-on.
+ * Configuration options for CloudWatch Adot add-on.
  */
 export interface CloudWatchAdotAddOnProps {
```

Comment:
Let's add dimensions here.

Reply:
Done.

Reply:
Other than EKS_Cluster, EKS_Namespace and EKS_PodName are there any other dimensions that customer will be interested in? I see metrics selectors but as it stands dimensions are fixed.